packages/use-i18n/src/usei18n.tsx

```diff
@@ -390,14 +393,31 @@ const I18nContextProvider = ({
         return ''
       }
       if (context) {
-        return formatters
-          .getTranslationFormat(value, currentLocale)
-          .format(context) as string
+        try {
+          return formatters
+            .getTranslationFormat(value, currentLocale)
+            .format(context) as string
+        } catch (err) {
+          onTranslateError?.(err as Error)
```

Comment:
Sounds great, may be we could provide `currentLocale` as string  to quickly find which locale file the is related  to the issue

Reply:
Exact, something in mind https://github.com/scaleway/scaleway-lib/pull/1747